src/Pim/Bundle/CatalogBundle/EventSubscriber/IndexProductsSubscriber.php

```diff
@@ -2,13 +2,12 @@
 
 namespace Pim\Bundle\CatalogBundle\EventSubscriber;
```

Comment:
I don't think because this subscriber is not huge, we could use it to put everything related to ES indexation for now:)